src/app/component/market/market-position-detail.tsx

```diff
@@ -0,0 +1,105 @@
+"use client";
+import Image from "next/image";
+import { useRouter } from "next/navigation";
+import { IoMdArrowBack } from "react-icons/io";
+import etheriumIcon from "../../../../public/icons/etherium.svg";
+
+export const MarketPositionDetail = () => {
+  const router = useRouter();
+
+  return (
+    <section className="">
+      <div>
+        <button onClick={() => router.push("/market")}>
+          <IoMdArrowBack className="hover:text-appColor1 h-5 w-5" />
+        </button>
```

Comment:
We are not using the common buttons here?  each button is having different type of style

Reply:
I will fix this in next PR